src/lualib/ArraySplice.ts

```diff
@@ -1,6 +1,11 @@
-function __TS__ArraySplice<T>(this: void, list: T[], start: number, deleteCount: number, ...items: T[]): T[] {
+// https://www.ecma-international.org/ecma-262/9.0/index.html#sec-array.prototype.splice
+function __TS__ArraySplice<T>(this: void, list: T[], ...args: Vararg<unknown>): T[] {
     const len = list.length;
 
+    const actualArgumentCount = select("#", ...args);
+    const start = select(1, ...args) as number;
```

Comment:
Can't `start` be a regular argument? Though if it's just to keep `args` name semantics I'm fine with that

Reply:
It cannot because of line 22, the ECMAScript spec prescribes a check 'if no arguments are provided', which includes `start`